server/controllers/File.ts

```diff
@@ -121,26 +124,16 @@ export default class FileController {
         const transaction = await this._database.sequelize.transaction();
 
         try {
-            // Если пользователь не передан при установке/смене аватара на своей странице - выдаем ошибку
-            if (!req.user) {
-                await transaction.rollback();
-                return res.status(HTTPStatuses.NotFound).send({ success: false, message: ErrorTextsApi.USER_NOT_FOUND });
-            }
-
             const userId = (req.user as IUser).id;
             const fileUrl = (req as IRequestWithSharpData).sharpImageUrl;
             const dublicateFileUrl = (req as IRequestWithSharpData).dublicateSharpImageUrl;
 
-            if (!userId) {
-                throw "Не найден идентификатор пользователя";
-            }
-
```

Comment:
You remove the conditions "if" with req.user & userId because it is File controller and here we use  Single-responsibility principle ?